Source/Frontend/StandaloneRTC/StandaloneRTC.csproj

```diff
@@ -28,29 +17,27 @@
     <IsWebBootstrapper>false</IsWebBootstrapper>
     <UseApplicationTrust>false</UseApplicationTrust>
     <BootstrapperEnabled>true</BootstrapperEnabled>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <UseWindowsForms>true</UseWindowsForms>
```

Comment:
There might be an issue where form designer doesn't open in projects that have converted to the new format. See https://stackoverflow.com/questions/53487757/how-to-open-form-designer-in-visual-studio

Until I can figure this out, I'm making this change a draft. I consider this a blocking issue.

Reply:
Nevermind, this is only happening in the RTC-Launcher project